src/pages/components/draggable.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Draggable
+description: >
+  The draggable is an interactive element that can be "grabbed", dragged, and dropped to
+  another location. This lets users rearrange and/or organize items.
+package: '@zendeskgarden/react-drag-drop'
+components:
+  - drag-drop/src/elements/draggable/Draggable.tsx
+  - drag-drop/src/elements/draggable-list/DraggableList.tsx
+  - drag-drop/src/elements/dropzone/Dropzone.tsx
+subcomponents:
+  - drag-drop/src/elements/draggable/components/Content.tsx
+  - drag-drop/src/elements/draggable/components/Grip.tsx
+  - drag-drop/src/elements/draggable-list/components/DropIndicator.tsx
+  - drag-drop/src/elements/draggable-list/components/Item.tsx
+  - drag-drop/src/elements/dropzone/components/Icon.tsx
+  - drag-drop/src/elements/dropzone/components/Message.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>Reordering a list</li>
+      <li>Moving objects between lists</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To let users upload files, use [File upload](/components/file-upload) instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+<!-- markdownlint-disable no-duplicate-header -->
+
+## How to use it
+
+### Default
+
+The Draggable has a content area and a grip to communicate it can be dragged and dropped.
+
+import DraggableDefault from '../../examples/draggable/DraggableDefault.tsx';
+import DraggableDefaultCode from '!!raw-loader!../../examples/draggable/DraggableDefault.tsx';
+
+<CodeExample code={DraggableDefaultCode}>
+  <DraggableDefault />
+</CodeExample>
+
+### Bare
+
+Borders can be removed using `isBare` prop.
+
+import DraggableBare from '../../examples/draggable/DraggableBare.tsx';
+import DraggableBareCode from '!!raw-loader!../../examples/draggable/DraggableBare.tsx';
+
+<CodeExample code={DraggableBareCode}>
+  <DraggableBare />
+</CodeExample>
+
+### Content
+
+The Draggable content area can accept other components and text.
+
+import DraggableContent from '../../examples/draggable/DraggableContent.tsx';
+import DraggableContentCode from '!!raw-loader!../../examples/draggable/DraggableContent.tsx';
+
+<CodeExample code={DraggableContentCode}>
+  <DraggableContent />
+</CodeExample>
+
+### Disabled
+
+The Draggable can be disabled to prevent interaction. It doesn't appear in the tab order,
+can't receive focus, and may not read aloud by a screenreader.
```

Comment:
Just noticed, the wording is missing something.
```suggestion
can't receive focus, and may not be read aloud by a screenreader.
```

Reply:
Thank you my good sir.